include/votca/xtp/moldenwriter.h

```diff
@@ -0,0 +1,70 @@
+/*
+ *            Copyright 2009-2020 The VOTCA Development Team
+ *                       (http://www.votca.org)
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License")
+ *
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+#ifndef VOTCA_XTP_MOLDEN_WRITER_H
+#define VOTCA_XTP_MOLDEN_WRITER_H
+
+// Local VOTCA includes
+#include "votca/xtp/logger.h"
+#include "votca/xtp/orbreorder.h"
+#include "votca/xtp/qmtool.h"
+#include <fstream>
+#include <votca/xtp/orbitals.h>
+
+namespace votca {
+namespace xtp {
+class MoldenWriter {
+ public:
+  MoldenWriter(Logger& log) : _log(log){};
+
+  ~MoldenWriter() = default;
+
+  void WriteFile(const std::string& filename, const Orbitals& orbitals);
+
+ private:
+  // clang-format off
+  Logger& _log;
+  std::array<Index,25> _multipliers={
```

Comment:
Seems to be the same as in the reader....

Reply:
actually yes, the reader and writer could be functions of a molden class.

What do you think @rubengerritsen .

Reply:
Initially I thought I had a good reason to keep them separate, but I can't think of it anymore. I have combined them now.